src/sentry/api/bases/project.py

```diff
@@ -78,6 +78,15 @@ class ProjectAlertRulePermission(ProjectPermission):
     }
 
 
+class ProjectOwnershipPermission(ProjectPermission):
+    scope_map = {
+        "GET": ["project:read", "project:write", "project:admin"],
+        "POST": ["project:write", "project:admin"],
+        "PUT": ["project:read", "project:write", "project:admin"],
```

Comment:
Add `project:read` to the scope map for `PUT` for `ProjectOwnership` to allow project members to make PUT requests

Reply:
we might put this in `src/sentry/api/endpoints/project_ownership.py` if its named ProjectOwnershipPermission

Reply:
Yeah, I considered that too, but currently there are a bunch of other project related permissions that are also defined here and imported for use elsewhere (`ProjectSettingPermission`, `ProjectAlertRulePermission`, etc.)